### PR TITLE
declarative_net_request availability

### PR DESCRIPTION
**Pull request by rebloor**
#### Summary

Update for the availability of `declarative_net_request` in Firefox 113

### CODE REVIEW

**Review by Rob--W — CHANGES_REQUESTED** [review]
Line 87: (pre-existing issue) remove L at the end of the URL:
https://github.com/mdn/browser-compat-data/blob/677986b1701355d50450e3741cb05016a5c00621/webextensions/api/declarativeNetRequest.json#L87

Line 748: RuleAction.requestHeaders.operation firefox version_added 113 instead of false
https://github.com/mdn/browser-compat-data/blob/677986b1701355d50450e3741cb05016a5c00621/webextensions/api/declarativeNetRequest.json#L748

Line 767: RuleAction.requestHeaders.operation.value firefox version_added 113 instead of false
https://github.com/mdn/browser-compat-data/blob/677986b1701355d50450e3741cb05016a5c00621/webextensions/api/declarativeNetRequest.json#L767

Line 1618: testMatchOutcome.options.includeOtherExtensions firefox version_added 113 instead of false
https://github.com/mdn/browser-compat-data/blob/677986b1701355d50450e3741cb05016a5c00621/webextensions/api/declarativeNetRequest.json#L1618

Mark the three DNR permissions as supported at
https://github.com/mdn/browser-compat-data/blob/677986b1701355d50450e3741cb05016a5c00621/webextensions/manifest/permissions.json

Add the DNR permissions at https://github.com/mdn/browser-compat-data/blob/677986b1701355d50450e3741cb05016a5c00621/webextensions/manifest/optional_permissions.json, with support state (Firefox/Chrome) according to https://bugzilla.mozilla.org/show_bug.cgi?id=1827855. I have asked Apple for support status on Safari, I'll get back to you when I have an update.

File: webextensions/manifest/optional_permissions.json [hunk 1]
```diff
@@ -272,6 +272,67 @@
...
+              "safari_ios": "mirror"
+            }
+          }
+        },
+        "declarativeNetRequestWithHostAccess": {
+          "__compat": {
+            "description": "<code>declarativeNetRequestWithHostAccess</code>",
+            "support": {
+              "chrome": {
+                "version_added": "84"
```
Comment (Rob--W) [inline webextensions/manifest/optional_permissions.json:321]:
```suggestion
                "version_added": "96"
```

Permission was introduced in 95 -  https://chromium.googlesource.com/chromium/src.git/+/2bfaa6f0446e2fc82485c53e351e1e835a0a2edc for the commit, and VERSION of that commit: https://chromium.googlesource.com/chromium/src.git/+/2bfaa6f0446e2fc82485c53e351e1e835a0a2edc/chrome/VERSION

And finally enabled for (stable) release in 96 through https://chromium.googlesource.com/chromium/src.git/+/a3d03e2823ab6c431e8e56ac10d5abac12881e09 - with the following VERSION: https://chromium.googlesource.com/chromium/src.git/+/a3d03e2823ab6c431e8e56ac10d5abac12881e09/chrome/VERSION

**Review by Rob--W — CHANGES_REQUESTED** [review]
> Line 87: (pre-existing issue) remove L at the end of the URL:
> https://github.com/mdn/browser-compat-data/blob/677986b1701355d50450e3741cb05016a5c00621/webextensions/api/declarativeNetRequest.json#L87

I don't see this change in the diff.

**Comment by rebloor** [comment]
@Rob--W hopefully now just waiting on your feedback from Apple

**Review by Rob--W — APPROVED** [review]
Let's merge this. I've heard back from Apple that the permissions can be optional, though there are some bugs with it. If more specific information is desired for Safari, a follow-up PR can be created later.

**Comment by rebloor** [comment]
Thanks @Rob--W - Done

**Pull request merged**